administration/src/util/translations/de.json

```diff
@@ -599,7 +599,7 @@
     "invalidApplicationConfirmationNoteSize": "Unzulässige Zeichenlänge der Notiz erreicht. Maximal sind {{maxSize}} Zeichen erlaubt.",
     "entitlementNotFound": "Wir konnten Ihre Angaben nicht im System finden.\nAus technischen Gründen stehen die Daten des Jobcenters Koblenz ab Mitte des folgenden Monats zur Verfügung.\nBitte überprüfen Sie Ihre Angaben und versuchen Sie es erneut.",
     "entitlementExpired": "Sie sind nicht länger berechtigt, einen KoblenzPass zu erstellen. Bitte kontaktieren Sie koblenzpass@stadt.koblenz.de für weitere Informationen.",
-    "mailNotSend": "Email konnte nicht an {{recipient}} gesendet werden.",
+    "mailNotSend": "E-Mail konnte nicht an {{recipient}} gesendet werden.\nBitte probieren Sie es zu einem späteren Zeitpunkt erneut.\nIhre Antragsdaten bleiben erhalten.",
```

Comment:
@connium maybe you got some better ideas here what to show. This error only occurs if our mail service is broken, just for some context